src/scenarios/init.ps1

```diff
@@ -1,2 +1,2 @@
-$scripts = Join-Path $PSScriptRoot '..\..\scripts\' -Resolve
+$scripts = Join-Path $PSScriptRoot '..\..\scripts' -Resolve
```

Comment:
just wondering is it more like a nit thing to remove the slash or otherwise it would break

Reply:
just a nit thing. I don't think it breaks anything.